app/composer.json

```diff
@@ -54,7 +54,6 @@
     "symfony/twig-bundle": "~4.4.0",
     "symfony/doctrine-bridge": "~4.4.0",
     "symfony/monolog-bridge": "~4.4.0",
-    "symfony/swiftmailer-bundle": "~3.3",
```

Comment:
It looks like Swiftmailer is not removed from composer.lock. Only here.

Reply:
I updated composer.lock, it should be removed by noew

Reply:
I just noticed that we should also remove packages like Sparkpost, sendgrid and others that are used to send emails. As those will be moved to separate plugins.